docs/plans/b21-edition-2024-migration-plan.md

```
@@ -0,0 +1,105 @@
+`docs/research/b21-edition-2024-migration.md` + the verified site list below ARE
+the scaffold; the flip is a focused follow-up that adds the macOS run.
+
+## Verified flagged sites (real lint output, not estimate)
+
+`RUSTFLAGS="-W tail_expr_drop_order -W if_let_rescope" cargo check
+--no-default-features --features cloud` in WSL (Linux, 2026-05-31) →
+**`audio-graph (lib) generated 28 warnings`**; the distinct audio-graph
+`tail_expr_drop_order`/`if_let_rescope` sites (cloud build; default/local-ml +
+macOS add more):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)